src/components/seo/index.js

```diff
@@ -27,16 +27,16 @@ const SEO = ({ description, og, imageUrl, title, titleTemplate, twitter, url })
 
 SEO.propTypes = {
   /**
-   * Meta description. Serves as a backup for og:desciption and
-   * twitter:desciption.
+   * Meta description. Serves as a backup for og:description and
+   * twitter:description.
```

Comment:
Haha — thank you! I made this change upstream, too (after installing a spell checker). Woops!